packages/xo-server/src/sensitive-values.js

```diff
@@ -28,7 +28,10 @@ export const obfuscate = value => replace(value, OBFUSCATED_VALUE)
 
 const SENSITIVE_PARAMS = {
   __proto__: null,
+  chapPassword: true,
   cifspassword: true,
+  newPassword: true,
+  oldPassword: true,
```

Comment:
Maybe we should obfuscate all keys containing the string `password`?

Reply:
I don't see an edge case or a risque for not doing it :+1: